### DISCUSSION TITLE
InnerSource Commons の日本語公式ページ作成

### DISCUSSION THREAD

**Discussion by yuhattor** [InnerSourceCommons/jp-roadmap#15] [Announcements]
InnerSource Learning Path は国際化されていますが、InnerSource Commons 自体も日本語化できないでしょうか。

**Comment by ystk** [comment]
https://innersourcecommons.org/ja/
↑言語関係の設定がパラメータ化されてるようなので、このURLの下に作れそうですよね。
リポジトリは、↓これかな。
https://github.com/InnerSourceCommons/innersourcecommons.org
